modules/libarchive/3.7.4.bcr.1/presubmit.yml

```diff
@@ -5,7 +5,7 @@ bcr_test_module:
       - debian10
       - debian11
       - ubuntu2004
-      # - ubuntu2004_arm64
+      - ubuntu2004_arm64
```

Comment:
We don't have good support for this platform on Bazel CI, please remove it.

Reply:
@meteorcloudy can we get some sort of linux arm platform that is supported? A lot of us are deploying software to Graviton these days for cost efficiency reasons and it's unfortunate to not have it tested upstream; especially for C code it can have a lot of architecture-specific bits